tsconfig.json

```diff
@@ -16,7 +16,8 @@
       "@clr/angular": ["src/clr-angular"],
       "@clr/icons": ["src/clr-icons"],
       "@clr/icons/shapes/*": ["src/clr-icons/shapes", "src/clr-icons/shapes/*"],
-      "@clr/base": ["src/clr-base", "src/clr-base/*"]
+      "@clr/base": ["src/clr-base"],
+      "@clr/base/*": ["src/clr-base", "src/clr-base/*"]
```

Comment:
Should there be two of these?

Reply:
Yes, this allows TS to map to the root `@clr/base` path as well as defined deep paths.
This is the default the CLI also generates. If we move `clr-angular` to follow the CLI practice of pointing to dist at dev time it would be similar.